### PR TITLE
#15_overlapping-ui-contents: remove overlaps

### LINKED ISSUE #15: Overlapping UI contents .
# Describe the bug
Trying out the web version of flutter , I accidently messed up the UI with the mobile version , as a result the content and text are overlapping .  

# To Reproduce

1 .Open the app
2. Navigate through some articles 
3. See bug

# Expected Behavior
Fix this bug . Contents should not overlap .  Accordingly adjust the size and padding . 

Screenshot_2020-10-12-23-58-48-286_com projects game mobile news_retic

### PR DESCRIPTION
**Pull request by davidfranquet**
Hope it works well 😄 

Simulator Screen Shot - iPhone SE (2nd generation) - 2020-10-13 at 14 32 01

### CODE REVIEW

File: lib/helper/ArticleData.dart [hunk 1]
```diff
@@ -22,127 +21,109 @@ class _ArticleDataState extends State<ArticleData> {
...
-                                          fontWeight: FontWeight.w400),
-                                    ),
+      child: ListView.builder(
+        physics: const AlwaysScrollableScrollPhysics(),
+        itemCount: widget.data.length,
+        itemBuilder: (context, index) {
+          return GestureDetector(
+            onTap: () {
+              mediaQuery.size.width > 700
+                  ? _launchURL(widget.data[index]['url'])
```
Comment (SAMYAK99) [inline lib/helper/ArticleData.dart]: You can remove this ternary operator . This was supposed to be used with the web version .

File: lib/helper/ArticleData.dart [hunk 2]
```diff
@@ -22,127 +21,109 @@ class _ArticleDataState extends State<ArticleData> {
...
+                          index,
+                        ),
+                      ),
+                    );
+            },
+            child: Container(
+              margin: EdgeInsets.only(
+                bottom: 20,
+                left: 4,
+                right: 4,
```
Comment (SAMYAK99) [inline lib/helper/ArticleData.dart]: Remove left and right margin . This was supposed to be used with the web version

**Review by SAMYAK99 — CHANGES_REQUESTED** [review]

File: lib/helper/ArticleData.dart (hunk 1 above)
Comment (davidfranquet) [reply lib/helper/ArticleData.dart]: done

File: lib/helper/ArticleData.dart (hunk 2 above)
Comment (davidfranquet) [reply lib/helper/ArticleData.dart]: done

**Comment by SAMYAK99** [comment]
Great work @DavidFD005  ! 
Keep Hacking !

**Review by SAMYAK99 — COMMENTED** [review]
Reviewed !

**Review by SAMYAK99 — COMMENTED** [review]
Review passed

**Pull request merged**